packages/table/src/TableWithVS.story.tsx

```diff
@@ -40,11 +40,12 @@ export default {
   // This is needed as a workaround to make arg spreading performant
   // https://github.com/storybookjs/storybook/issues/11657
   parameters: {
+    default: 'Basic',
```

Comment:
Is this necessary here? The website LiveExample currently is just served from `Table.story.tsx`.